feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,326 @@
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.data.Field;
+import com.esri.arcgisruntime.data.FieldDescription;
+import com.esri.arcgisruntime.data.Geodatabase;
+import com.esri.arcgisruntime.data.GeodatabaseFeatureTable;
+import com.esri.arcgisruntime.data.TableDescription;
+import com.esri.arcgisruntime.geometry.GeometryEngine;
+import com.esri.arcgisruntime.geometry.GeometryType;
+import com.esri.arcgisruntime.geometry.Multipoint;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.PointCollection;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.data.QueryParameters;
+
+public class CreateMobileGeodatabaseController {
+
+  @FXML private MapView mapView;
+  @FXML private Label label;
+  @FXML private Button viewTableButton, createGeodatabaseButton, closeGeodatabaseButton;
+  @FXML private Stage tableStage;
+
+  private ArcGISMap map;
+  private GraphicsOverlay graphicsOverlay;
+  private Geodatabase geodatabase;
+  private Path geodatabasePath;
+  private GeodatabaseFeatureTable geodatabaseFeatureTable;
+  private boolean isTableWindowOpen;
+
+  @FXML
+  private void initialize() {
+
+    try {
+      // authentication with an API key or named user is required to access base maps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the topographic imagery basemap style
+      map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // set the map to the mapview
+      mapView.setMap(map);
+
+      // create a point located at Redlands, CA to be used as the viewpoint for the map
+      var point = new Point(-117.195800, 34.056295, SpatialReferences.getWgs84());
+      mapView.setViewpointCenterAsync(point, 10000);
+
+      // create a graphics overlay to display the input points
+      graphicsOverlay = new GraphicsOverlay();
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a graphic to add the simple marker symbol in the graphics overlay
+      var simpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFF000000, 10);
+      var graphic = new Graphic();
+      graphic.setSymbol(simpleMarkerSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+
+      // keep track of the points added by the user
+      List<Point> inputs = new ArrayList<>();
+
+      // create a point from user input and add it to the geodatabase feature table
+      mapView.setOnMouseClicked(e -> {
+        if(createGeodatabaseButton.isDisabled() && !isTableWindowOpen) {
```

Comment:
Java styling best practice here with space as below
```suggestion
        if (createGeodatabaseButton.isDisabled() && !isTableWindowOpen) {
```